runelite-client/src/main/java/net/runelite/client/plugins/woodcutting/WoodcuttingPlugin.java

```diff
@@ -107,6 +121,7 @@
 	private final List<TreeRespawn> respawns = new ArrayList<>();
 	private boolean recentlyLoggedIn;
 	private int currentPlane;
+	ClueNestTier clueTierSpawned = null;
```

Comment:
`null` is the default value for object fields.
```suggestion
	ClueNestTier clueTierSpawned;
```

Reply:
Also, this should be reset in shutDown or startUp